src/providers/twitch/api/Helix.cpp

```diff
@@ -2468,6 +2468,100 @@ void Helix::startCommercial(
         .execute();
 }
 
+// Twitch global badges
+// https://dev.twitch.tv/docs/api/reference/#get-global-chat-badges
+void Helix::getGlobalBadges(
+    ResultCallback<HelixGlobalBadges> successCallback,
+    FailureCallback<HelixGetGlobalBadgesError, QString> failureCallback)
+{
+    using Error = HelixGetGlobalBadgesError;
+    this->makeRequest("chat/badges/global")
+        .type(NetworkRequestType::Get)
+        .header("Content-Type", "application/json")
```

Comment:
```suggestion
```

Reply:
Thanks for the correction! Fixed in the last commit